### PR TITLE
docs(components): Improve Atlantis branding on Storybook

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations
Change the `a` tags colors and use our own `Heading` and `Text`.

### Note
This will temporarily show up since the existing readme needs it for docz

image

https://github.com/GetJobber/atlantis/blob/c30d262b2be0c862ee349f7a13ec63ce615f1e30/docs/README.md?plain=1#L1-L4

## Changes
- Change the `a` tags colors and use our own `Heading` and `Text`.

### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -8,4 +15,14 @@ export const parameters = {
       order: ["Components", "Design", "*"],
     },
   },
-}
+  docs: {
+    components: {
+      wrapper: props => <Content>{props.children}</Content>,
+      h1: props => <Heading {...props} level={1} />,
```
Comment (chris-at-jobber) [inline .storybook/preview.js:21]: Does this break the clickable/copyable heading URLs? @darryltec mentioned that might be the reason those don't work in our mobile Storybook instance.

Comment (darryltec) [reply .storybook/preview.js:21]: most likely, i'll bring em back on my PR. Seems like we have to do some custom bits on it to make it play nice

Comment (chris-at-jobber) [reply .storybook/preview.js:21]: We could potentially also just drop the CSS into the `preview-head` 🤷 

I think the crucial adjustments would be

`{ color: var(--color-heading); font-family: var(--typography--fontFamily-display); }`

Comment (darryltec) [reply .storybook/preview.js:21]: They use semantic tags anyway so we probably could just target those. The alternative is yeah, implement our own link anchor

**Comment by chris-at-jobber** [comment]
Seeing discrepancies across pages:

❌  **Intro** 
image

✅  **Components**
image

✅ **Design**
image

✅ **Basically any other page**
image

**Comment by chris-at-jobber** [comment]
I just saw that PR allowing `.md` to be imported so I'm assuming that's the reason for the difference on the ReadMe 😄 - can we resolve that?

**Comment by chris-at-jobber** [comment]
I also noticed that the links in the content are `interactive` but the links in the TOC sidebar are not - is that something we can update?

image

**Comment by chris-at-jobber** [comment]
One last piece / maybe separate from this work, the shadow from the "collapsed" TOC sidebar is visible on narrow viewports:

image

**Comment by darryltec** [comment]
@chris-at-jobber those thangs you saw should be fixed now

**Comment by chris-at-jobber** [comment]
I see this note in the description - how temporary is this? 

image

If we haven't already, we need to add an explicit note to the followup ticket that removes Docz to clean this up

**Review by chris-at-jobber — COMMENTED** [review]
Design approved ✅

**Comment by rodrigoeidelvein** [comment]
I know that I'm the PR creator, but @darryltec took over this, LGTM!

**Review by darryltec — APPROVED** [review]
Rodrigo's changes also looks good!

**Pull request merged**